src/ng/httpBackend.js

```diff
@@ -105,14 +105,22 @@ function createHttpBackend($browser, createXhr, $browserDefer, callbacks, rawDoc
             statusText);
       };
 
+      var isTimedOut = false;
       var requestError = function() {
         // The response is always empty
         // See https://xhr.spec.whatwg.org/#request-error-steps and https://fetch.spec.whatwg.org/#concept-network-error
-        completeRequest(callback, -1, null, null, '');
+        completeRequest(callback, -1, null, null, 'Error');
```

Comment:
Discussed offline, we cannot change error code since that would require changing the code throughout that does handling based on -1.

Changed to UnknownError as per suggestion.